### PR TITLE
Run the protoc command

### PR DESCRIPTION
**Pull request by drewroengoogle**
This came from running `protoc --dart_out=. lib/src/model/proto/**/*.proto` as per the [README](https://github.com/flutter/cocoon/blob/main/app_dart/README.md#protobuf).

This allows ci.yaml to run successfully when setting a scheduler to `release`

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read the [Flutter Style Guide] _recently_, and have followed its advice.
- [x] I signed the [CLA].
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

**Review by CaseyHillers — COMMENTED** [review]
Nice catch! I had a githook for this, and forgot this was happening.

Can you add it to https://github.com/flutter/cocoon/blob/main/test_utilities/bin/dart_test_runner.sh to ensure the generated proto code is up to date?

File: test_utilities/bin/dart_test_runner.sh [hunk 1]
```diff
@@ -14,6 +14,10 @@ pushd "$1" > /dev/null
 flutter clean
 dart pub get
 
+echo "############# update proto generated code #############"
+protoc --dart_out=. lib/src/model/proto/**/*.proto
```
Comment (CaseyHillers) [inline test_utilities/bin/dart_test_runner.sh:18]: This only runs the formatter on the bot. It also needs something like `git diff --exit-code` to fail if there are local changes

Comment (drewroengoogle) [reply test_utilities/bin/dart_test_runner.sh]: Okay, that makes sense now that I look at it. Thanks! Will update.

File: test_utilities/bin/dart_test_runner.sh [hunk 2]
```diff
@@ -14,10 +14,11 @@ pushd "$1" > /dev/null
 flutter clean
 dart pub get
 
-echo "############# update proto generated code #############"
-apt-get install protocol-compiler
-protoc --dart_out=. app_dart/lib/src/model/proto/**/*.proto
-echo "#######################################################"
+# TODO(drewroen): Validate proto code has been generated.
```
Comment (CaseyHillers) [inline test_utilities/bin/dart_test_runner.sh:17]: https://dart-lang.github.io/linter/lints/flutter_style_todos.html

Comment (drewroengoogle) [reply test_utilities/bin/dart_test_runner.sh]: Fixed! Feel free to resolve.

Comment (CaseyHillers) [reply test_utilities/bin/dart_test_runner.sh]: It's still missing. Format should be:

TODO(username): Short description. https://github/issue/link

Comment (drewroengoogle) [reply test_utilities/bin/dart_test_runner.sh]: Sorry about that. I thought the issue was with the incorrect username. I've added an additional issue link.

**Review by CaseyHillers — APPROVED** [review]

**Pull request merged**